### ISSUE TITLE
Update gemspec to use rspec 3

### ISSUE THREAD

**Issue by toothfairy** [hyperoslo/tasuku#70]
Rspec 2 is old and bunk, Rspec 3 is new and cool

**Comment by sindrenm** [comment]
This is true! However, there are some backwards-incompatible things to handle. :smiley:

**Comment by toothfairy** [comment]
I hope we don't have any of them here, cause I already spent like 3 hours replacing should to expect syntax this week

**Comment by timkurvers** [comment]
Expect-syntax is supported in RSpec 2, too, no?

**Comment by toothfairy** [comment]
it is! but should syntax is deprecated in rspec 3
